### PR TITLE
Updated docs to better match default server config

### PR DESCRIPTION
**Pull request by Bruce-Hopkins**
**What is the purpose of this pull request? (put an "X" next to item)**

[X ] Documentation update
[ ] Bug fix
[ ] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
Updated server docs. Changed to fix syntax errors and also better match the default config for the server.

**Which issue (if any) does this pull request address?**
#403 

**Is there anything you'd like reviewers to focus on?**
n/a

### CODE REVIEW

**Comment by Bruce-Hopkins** [comment]
Are the changes okay for merging? @DiegoRBaquero

**Comment by DiegoRBaquero** [comment]
We don't use var anymore. Also needs to match standard format

**Comment by Bruce-Hopkins** [comment]
@DiegoRBaquero I made changes from `var` to `const`. Does this work?

File: README.md [hunk 1]
```diff
@@ -230,12 +230,34 @@ server.on('warning', function (err) {
...
+  const wsPort = wsAddr.port
+  console.log(`WebSocket tracker: ws://${wsHost}:${wsPort}`)
+
 })
 
+
 // start tracker server listening! Use 0 to listen on a random free port.
-server.listen(port, hostname, onlistening)
+const port = 0,
+hostname = "localhost"
```
Comment (DiegoRBaquero) [inline README.md:257]:
```suggestion
const port = 0
const hostname = "localhost"
```

**Review by DiegoRBaquero — APPROVED** [review]

**Pull request merged**